### DISCUSSION TITLE
用AI取代菜鳥設計師寫程式，也許比你預期的更快！AlphaCode已與一般程式設計師能力相當

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#518] [General]
### 用AI寫程式，不只一家在研究
這篇文章討論了一個名為AlphaCode的人工智慧（AI）新的前端應用，它是由卡內基梅隆大學的研究人員開發的。AlphaCode是一個使用大型語言模型的程式，根據問題的自然語言描述來建構程式碼。該軟體利用了程式問題和解決方案的資料集，加上GitHub上的一組非結構化程式碼，以學習電腦程式的結構。研究人員發現，AlphaCode在程式風格和質量方面均超過了其他AI工具，研究突顯了DeepMind在大型語言模型或LLM領域的領先地位。
https://www.techbang.com/posts/102423-the-next-frontier-in-artificial-intelligence-alphacode-is

**Comment by NcuJerryKao** [comment]
什麼 幾年前看AI將取代的職業 工程師不在當中的說

**Comment by williamyen0623** [comment]
完了，芭比q了。難道連工程師都要被AI取代了嗎？！

**Comment by BingRong1217** [comment]
連程式都可以交給AI了，還有什麼他們做不到的嗎

**Comment by brian431** [comment]
各位放心啦，ai 要取代的工作工程師肯定不是最先的幾個

**Comment by xCk27x** [comment]
「競爭性程式是一項極其困難的挑戰，我們現在的水準（在10次提交中解決約30%的問題）與頂級程式設計師（在一次提交中解決>90%的問題）之間存在巨大差距，」
目前這種自然語言處理的coding AI還是跟頂端的程式設計師有蠻大的差距
這代表各位要繼續繼續加油努力了!!

**Comment by Aerofe** [comment]
這樣也可以促進每個工程師精進自己，不然還真的有可能被AI取代

**Comment by Yilin20021016** [comment]
AI is extremely good but dangerous if we cannot control it.

**Comment by HsuChen88** [comment]
如果菜鳥都用AI寫，那要何時才能變不菜

**Comment by larry40305** [comment]
不管什麼工作都會被取代，就連工程師也會被AI取代
